atlp-rwanda/e-commerce-lydia-32-bn
# Payment with stripe

### What does this PR do?
This PR implements payment processing functionality using Stripe for our e-commerce application. It includes the integration of Stripe to handle payment intents, secure payment processing, and confirmation of payments from the frontend to the backend.

### Description of Task to be completed?
Install and configure Stripe library in the backend.
Create an endpoint to generate payment intents.
Handle payment confirmation and error handling on the frontend.
Add documentation and example usage for the payment processing feature.

### How should this be manually tested?
Clone the repository and checkout to this PR's branch.
Install necessary dependencies using npm install.
Start the server using npm start.
Use Postman or Swagger doc to test the /create-payment endpoint by sending a POST request with a JSON body containing the amount and currency.
Verify that a payment intent is created and a client secret is returned.

### What are the relevant pivotal tracker stories
https://www.pivotaltracker.com/story/show/187570888
